### DISCUSSION TITLE
Documentation Update - Firewall Requirements

### DISCUSSION THREAD

**Discussion by TheRealPancakes** [Security-Onion-Solutions/securityonion#13108] [2.4] [answered]
### Version

2.4.70

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

48

### RAM

512G

### Storage for /

445.07 GiB

### Storage for /nsm

11.64 TiB

### Network Traffic Collection

tap

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

The [Internet Communication](https://docs.securityonion.net/en/2.4/firewall.html#internet-communication) section of documentation should be updated to indicate that access to `objects.githubusercontent.com` is also required for "Sigma rules updates". This has been confirmed via proxy log audit.

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by TOoSmOotH** [comment, marked as answer]
Good catch. We will work on getting it added.

**Reply by dougburks** [reply]
Updated:
https://docs.securityonion.net/en/2.4/firewall.html#internet-communication
